echemdb/data/cv/entry.py

```diff
@@ -175,29 +175,29 @@ def format_title(self, e, which_field, as_sentence=True):
 
     def x(self):
         r"""
-        Return the name of the variable on the x-axis, i.e., `"U"`.
-
-        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+        Return the name of the variable on the x-axis, i.e., `"E"`.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.x()
-            'U'
+            'E'
 
         """
         from astropy import units as u
-        if u.Unit(self.figure_description.potential_scale.unit).is_equivalent('V'):
-            return 'U'
+
+        if not self.data_description.axes:
```

Comment:
I think this implementation is more complex than it has to be. The more straightforward implementation would be:

```
if not axes:
    raise
if 'E' not in:
    raise
if not equivalent:
    raise # or why would you want to only assert this?

return 'E'
```

Reply:
@saraedum I hope this is what you wanted, as well as for `y` down below.

Reply:
I guess you can merge this PR if you agree.